docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+services:
+  db:
+    image: postgres
+    volumes:
+      - kelvin-postgres-data:/var/lib/postgresql/data
+    container_name: kelvin_db
+    environment:
+      - POSTGRES_DB=${DB_DATABASE}
+      - POSTGRES_USER=${DB_USERNAME}
+      - POSTGRES_PASSWORD=${DB_PASSWORD}
```

Comment:
we should also put variable for host:
```
- POSTGRES_HOST=db
```

and use these envs inside `kelvin/settings.py`:
```
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'HOST': os.getenv('POSTGRES_HOST'),
+        'USER': os.getenv('POSTGRES_USER'),
+        'PASSWORD': os.getenv('POSTGRES_PASSWORD'),
+        'NAME': os.getenv('POSTGRES_DB'),
     }
 }
```
because currently it uses sqlite :(